README.md

```diff
@@ -13,3 +13,11 @@ Travis:   [![Travis](https://travis-ci.org/aspnet/Localization.svg?branch=dev)](
 Localization abstractions and implementations for ASP.NET Core applications.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+### Samples
+
+Here are a few samples that demonstrate different localization features including: localized views, localized strings in data annotations, creating custom localization resources ... etc.
+
+ * [Localization.StarterWeb](https://github.com/aspnet/Entropy/tree/dev/samples/Localization.StarterWeb) - comprehensive localization sample demonstrates almost **all** of the localization features
```

Comment:
Sorry I didn't mean for you to include the `**` in the text 😄 It was just my emphasis in the comment to show what I changed. Sorry!